Cirrious.FluentLayout/FluentLayoutExtensions.cs

```diff
@@ -18,8 +18,8 @@ public static void SubviewsDoNotTranslateAutoresizingMaskIntoConstraints(this UI
             foreach (var subview in view.Subviews)
             {
                 subview.TranslatesAutoresizingMaskIntoConstraints = false;
-		
-	        if (subview.Subviews != null && subview.Subviews.Count() > 0)
+
+                if ((subview as UIControl) == null && subview.Subviews != null && subview.Subviews.Count() > 0)
```

Comment:
all system controls, that have some subviews, have a common superclass (UIControl).

eg:  `UIButton`, `UISwitch`, `UISegmentedControl`, `UISlider` and so on.

I just find an Constraint issue with `UISwitch` if without `(subview as UIControl) == null`